aptos-move/framework/aptos-framework/sources/Coin.move

```diff
@@ -263,6 +287,16 @@ module AptosFramework::Coin {
         }
     }
 
+    public fun initialize_coin_events(account: &signer) {
+        assert!(
+            !exists<CoinEvents>(Signer::address_of(account)),
+            Errors::not_published(ECOIN_EVENTS_ALREADY_PUBLISHED),
+        );
+        move_to(account, CoinEvents {
+            register_events: Event::new_event_handle<RegisterEvent>(account),
+        });
+    }
+
```

Comment:
just make this inline, no need to make it otherwise since it will inevitably be called upon introducing Coin<Test>